pkg/p2p/p2p.go

```diff
@@ -17,6 +17,7 @@ import (
 // Service provides methods to handle p2p Peers and Protocols.
 type Service interface {
 	AddProtocol(ProtocolSpec) error
+	ConnectNotify(ctx context.Context, addr ma.Multiaddr) (address *bzz.Address, err error)
```

Comment:
I think that comments on `Connect` and `ConnectNotify` methods would be very useful here to distinguish their usage.